src/network/networkd-dhcp6.c

```diff
@@ -992,8 +992,10 @@ static int dhcp6_update_address(
         addr->cinfo.ifa_valid = lifetime_valid;
 
         (void) in_addr_to_string(addr->family, &addr->in_addr, &buffer);
-        log_link_info(link, "DHCPv6 address %s/%u timeout preferred %d valid %d",
-                      strna(buffer), addr->prefixlen, lifetime_preferred, lifetime_valid);
+
+        if (!set_contains(link->dhcp6_addresses, addr))
```

Comment:
maybe use `log_link_full(link, set_contains(…) ? LOG_DEBUG : LOG_INFO)` or so?

Reply:
(i.e. just downgrade to debug log if it's an existing message, given it might still be interesting?

Reply:
Sorry for late response. Updated.